javascript-and-json/main.js

```diff
@@ -0,0 +1,32 @@
+/* eslint-disable quote-props */
+/* eslint-disable quotes */
+var booksArray = [
+  {
+    title: 'The Hobbit',
+    isbn: '0395485799',
+    author: 'J.R.R. Tolkien'
+  },
+  {
+    title: 'The Lord of the Rings',
+    isbn: '043935806X',
+    author: 'J.R.R. Tolkien'
+  },
+  {
+    title: 'The Catcher in the Rye',
+    isbn: '0312364563',
+    author: 'J.D. Salinger'
+  }
+];
+
+console.log('booksArray: ', booksArray, '\n', 'typeof booksArray: ', typeof booksArray);
+
+var booksString = JSON.stringify(booksArray);
+console.log('booksString: ', booksString, '\n', 'typeof booksString: ', typeof booksString);
+
+var studentsString = "{\"Number id\":\"123456789\",\"name\":\"Jeff McLean\"}";
```

Comment:
so instead of using escape characters, you want to use single quotes on the outside of the object, and double quotes for the properties and values